tritium-caffeine/src/main/java/com/palantir/tritium/metrics/caffeine/CaffeineCacheMetricSet.java

```diff
@@ -79,13 +79,15 @@ protected CacheStats loadValue() {
         };
     }
 
-    private String cacheMetricName(String... args) {
-        return MetricRegistry.name(MetricRegistry.name(cacheName, "cache"), args);
+    private MetricName cacheMetricName(String... args) {
+        return MetricName.builder()
+                .safeName(MetricRegistry.name("cache", args))
```

Comment:
do we actually know this will be safe?

Reply:
these are all defined statically. see https://github.com/palantir/tritium/pull/47/files/8f2eb5d14f7e7ad0ec09d67b24ffebf30af0edae#diff-080a8f778f65d5d6ee0d746b4395ca53R100 and below